core/quantity/src/test/DirectionFormatting.test.ts

```diff
@@ -0,0 +1,282 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { assert, expect } from "chai";
+import { Format } from "../Formatter/Format";
+import { FormatterSpec } from "../Formatter/FormatterSpec";
+import { Formatter } from "../Formatter/Formatter";
+import { TestUnitsProvider } from "./TestUtils/TestHelper";
+import { FormatProps, UnitProps } from "../core-quantity";
+
+describe.only("Bearing direction tests:", () => {
```

Comment:
remove this `only`

Reply:
I'll do a sweep of those once I'm finished with the changes.